README.md

```diff
@@ -22,14 +33,10 @@ This application has been build by the following developers:
 - Ludovic Dubost, XWiki SAS
 - Maxim Ionut, XWiki SAS
 - Anca Luca, XWiki SAS
+- Alexandru Cotiugă, XWiki SAS
```

Comment:
you should add this information in the pom, since the README is not the place for it (and when we will standardize the README format we might remove this part)